namespace-provisioner/customize-installation.md

```diff
@@ -48,39 +44,39 @@ Options if using Controller
 
    Sample TAP values configuration:
 
-   ```console
-   namespace_provisioner:
-     controller: true
-     additional_sources:
-     - git:
-         ref: origin/main
-         subPath: ns-provisioner-samples/testing-scanning-supplychain
-         url: https://github.com/vmware-tanzu/application-accelerator-samples.git
-         # secretRef section is only needed if connecting to a Private Git repo
-         secretRef:
-           name: git-auth
-           namespace: tap-install
-           create_export: true
-       path: _ytt_lib/testing-scanning-supplychain-setup
+   ```yaml
+  namespace_provisioner:
+    controller: true
+    additional_sources:
+    - git:
+        ref: origin/main
+        subPath: ns-provisioner-samples/testing-scanning-supplychain
+        url: https://github.com/vmware-tanzu/application-accelerator-samples.git
+        # secretRef section is only needed if connecting to a Private Git repo
+        secretRef:
+          name: git-auth
+          namespace: tap-install
+          create_export: true
+      path: _ytt_lib/testing-scanning-supplychain-setup
    ```
 
-   See [Git Authentication for Private repository](use-case3.md).
+   See [Git Authentication for Private repository](use-case3.md#git-private).
```

Comment:
this was referenced before, in line 38, should it go twice?

Reply:
We have this mentioned at two places in the referencing document as well